### ISSUE TITLE
Increase contrast on dividers between posts

### ISSUE THREAD

**Issue by Merlin04** [sk22/megalodon#911]
**Is your feature request related to a problem? Please describe.**
with the new UI update, I'm finding the dividers between posts to have substantially lower contrast than before, which makes it more difficult for me to skim posts. 

**Describe the solution you'd like**
An option to enable higher contrast for those dividers, or maybe a change to the default!

**Additional context**

![Screenshot_20231029-043733.png](https://github.com/sk22/megalodon/assets/11800751/e6e0981d-95dc-4e9c-876d-183260dfc594)

**Comment by Groxx** [comment]
Yes please. I mostly like the new styling, but it's now _extremely_ hard to tell if a reply is part of a thread or not. Particularly on some devices, since low-end contrast varies a _ton_.
